convert_state.py

```diff
@@ -1,60 +1,52 @@
+""" Program to convert state names to abbreviation and vice versa """
+import json
 
-
-
-import json  # import json for json processing
 def main():
     with open('data.json','r') as file:
         data = file.read()
         data = json.loads(data)
 
-    print(data)
+    # print(data)
     
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    state_Abbr = data # dictionary of state abbreviations keys and state name values 
+    state_Abbr2 = {} # dictionary of state name keys and state abbreviation values
     # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
-
+    for statekey, statevalue in state_Abbr.items() :
+        state_Abbr2[statevalue] =  statekey
 
-    #loop
     while True:
         print('1. Convert state to abbreviation')
-        print("2. Convery abreviasion to state")
-        print(' 3. quit')
-        choice =   input("Enter choice : ")
-
-        if choice=="1":
-
-            convertStateToAbbreviation(stateAbbr2)
-        elif  choice == "2":
-            convert_abbreviation_to_state(stateAbbr)
-        
-        
-        elif choice==  "3":
+        print('2. Convery abbreviation to state')
+        print('3. quit')
+        choice = input('Enter choice: ')
+
+        if choice=='1':
+            convert_state_to_abbreviation(state_Abbr2)
+        elif  choice == '2':
+            convert_abbreviation_to_state(state_Abbr)
+        elif choice==  '3':
             break
         else:
             print('try again')
-def convertStateToAbbreviation(dictionary ):
-    userInput   = input("Enter state name").  capitalize()
-    result = dictionary.get(userInput  )
-    if result == None:
-        print(' state not found')
+
+def convert_state_to_abbreviation(dictionary):
+    """ Allow user to get conversion from dictionary """
+    user_input = input('Enter state name').capitalize()
+    result = dictionary.get(user_input)
+    if result is None:
+        print('State not found')
     else:
-        print ( "The abbreviation for " +  userInput+" is " + result)
+        print ('The abbreviation for' +  user_input + ' is ' + result)
```

Comment:
While I don't think the linter catches this, I'd recommend changing these to an f string, as there's more than one variable being added to the string.